paypro-gateways-woocommerce/includes/paypro/wc/gateway.php

```diff
@@ -1,18 +1,24 @@
 <?php if(!defined('ABSPATH')) exit; // Exit if accessed directly
 
-abstract class PayPro_WC_Gateway_Abstract extends WC_Payment_Gateway
+class PayPro_WC_Gateway extends WC_Payment_Gateway
 {
-    protected $default_title;
+	/**
+	 * panther restapi responses
+	 */
+	protected const PAYPROAPIRES_NOTSUBSCRIPTED = "Not subscribed to money transfer service";
 
-    protected $default_description;
+    protected $default_title;
```

Comment:
Why do we keep this but remove `$default_description`? what about `$default_logo`

Reply:
no clue why they were actually there to begin with i removed them now and will be pushed in the next push